src/Features/Core/Portable/Completion/Providers/AbstractRecommendationServiceBasedCompletionProvider.cs

```diff
@@ -90,6 +95,19 @@ private ITypeSymbol GetSymbolType(ISymbol symbol)
                 supportedPlatforms: supportedPlatformData);
         }
 
+        private static bool ShouldSoftSelectInArgumentList(CompletionContext completionContext, SyntaxContext context, bool preselect)
+        {
+            return !preselect &&
+                completionContext.Trigger.Kind == CompletionTriggerKind.Insertion &&
+                context.IsOnArgumentListBracketOrComma &&
+                IsArgumentListTriggerCharacter(completionContext.Trigger.Character);
```

Comment:
doc.

Reply:
Which bit exactly?

Reply:
why this is acase where we want to soft-select.  this is for the person coming 2 years from now and saying "why?"